### PR TITLE
Adds a churn example.

### PR DESCRIPTION
**Pull request by jdoliner**

### CODE REVIEW

File: examples/churn/churn.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "cpu": 1
+    },
+    "egress": {
+        "sql_database": {
+            "url": "snowflake://albertpachyderm@CDKSVXM-PN23804/CHURN_DEMO/PUBLIC?warehouse=COMPUTE_WH",
+            "file_format": {
+                "type": "CSV"
+            },
+            "secret": {
+                "k8s_secret": "snowflakesecret",
```
Comment (albscui) [inline examples/churn/churn.json:31]:
```suggestion
                "name": "snowflakesecret",
```

**Review by albscui — APPROVED** [review]
LGTM, except we renamed `k8s_secret` to just `name`.

**Comment by albscui** [comment]
Do we have to upload `examples/churn/data.sql`? Seems like quite a large file. Is it possible to make it smaller?

**Comment by jdoliner** [comment]
> Do we have to upload `examples/churn/data.sql`? Seems like quite a large file. Is it possible to make it smaller?

I could slim it down a bit and the example would still work.

**Pull request merged**